src/app/workspace/resource/resource-instance-form/select-properties/select-properties.component.ts

```diff
@@ -17,7 +17,7 @@ export class SelectPropertiesComponent implements OnInit {
 
```

Comment:
I think now you should able to remove

```ts
export interface Properties {
    [index: string]: ResourcePropertyDefinition;
}
```

I think there is only an used import in `src/app/workspace/resource/resource-instance-form/select-properties/select-properties.component.spec.ts`